src/analyze/analyze-security.c

```diff
@@ -2245,12 +2251,114 @@ static int get_security_info(Unit *u, ExecContext *c, CGroupContext *g, Security
         return 0;
 }
 
-int analyze_security(sd_bus *bus, char **units, AnalyzeSecurityFlags flags) {
+static int offline_security_check(Unit *u) {
+        _cleanup_(sd_bus_error_free) sd_bus_error err = SD_BUS_ERROR_NULL;
+        Table *overview_table = NULL;
+        AnalyzeSecurityFlags flags = ANALYZE_SECURITY_ERROR_IF_EXPOSURE_ABOVE_MEDIUM;
+        _cleanup_(security_info_free_freep) SecurityInfo *info = NULL;
+        int r, k;
+
+        assert(u);
+
+        if (DEBUG_LOGGING)
+                unit_dump(u, stdout, "\t");
+
+        log_unit_debug(u, "Creating %s/start job", u->id);
+        r = manager_add_job(u->manager, JOB_START, u, JOB_REPLACE, NULL, &err, NULL);
```

Comment:
why enqueue a start job for th eunit? just load the unit, that's all you need

Reply:
that's how the `verify` verb implemented this, this is copied verbatim, iirc there's some things that are not loaded until then?

Reply:
verify tests more, i.e. whether the transaction actually works. but you don't want this here. you just the parsing, nothing more.